devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -67,27 +68,35 @@ def updated() {
 	updateDataValue("onOff", "catchall")
 }
 
-// Parse incoming device messages to generate events
 def parse(String description) {
 	log.debug "description is $description"
-	Map map = zigbee.getEvent(description)
-	if (map) {
-		if (description?.startsWith('on/off')) {
-			log.debug "receive on/off message without endpoint id"
-			sendHubCommand(refresh().collect { new physicalgraph.device.HubAction(it) }, 0)
+	Map eventMap = zigbee.getEvent(description)
+	log.debug "eventMap is $eventMap"
+	Map eventDescMap = zigbee.parseDescriptionAsMap(description)
+	log.debug "eventDescMap is $eventDescMap"
+
+	if (!eventMap && eventDescMap) {
+		log.debug "create new eventMap "
+		eventMap = [:]
+		if (eventDescMap?.cluster == "0006" || eventDescMap?.clusterId == "0006") {
+			eventMap[name] = "switch"
+			eventMap[value] = eventDescMap?.value
+		}
+	}
+
+	if (eventMap.value) {
+		if (eventDescMap?.endpoint == "01" || eventDescMap?.sourceEndpoint == "01") {
```

Comment:
I don't think `.endpoint` is a valid field.

Reply:
I noticed "endpoint" while i was trying to deal with setLevel for this device some time ago.